errors/no-cache.md

```diff
@@ -80,7 +80,9 @@ Using GitHub's [actions/cache](https://github.com/actions/cache), add the follow
 ```yaml
 uses: actions/cache@v2
 with:
-  path: ${{ github.workspace }}/.next/cache
+  path: |
+    ~/.npm
```

Comment:
I think this should be configured with `actions/setup-node` instead 

https://github.com/actions/setup-node#caching-packages-dependencies